src/framework/uicomponents/qml/Muse/UiComponents/internal/StyledMenuItem.qml

```diff
@@ -183,7 +183,7 @@ ListItemBlank {
         StyledIconLabel {
             id: primaryIconLabel
             Layout.alignment: Qt.AlignLeft
-            width: 16
+            Layout.preferredWidth: 20
```

Comment:
It looks like the original design files do hardcode this at 16. For some icons, this means that they will go a bit beyond the theoretical border of the StyledIconLabel, but that should be fine (and they are property centred).
```suggestion
            Layout.preferredWidth: 16
```
This goes for both the checkmark and the icon.

Reply:
I agree, 16 works too. I have changed both.